autoscheduler/scraper/management/commands/scrape_departments.py

```diff
@@ -0,0 +1,28 @@
+# Called to execute the beginning of scaping courses
+from django.core.management import base
+from scraper.banner_requests import BannerRequests
+from scraper.models import Department
+
+import time
+import datetime
+
+class Command(base.BaseCommand):
+    """ Gets all departments from banner and adds them to the database """
+
+    def scrape_departments():
+        term = add_arguments()x
+        var request = BannerRequests(term)
+        depts = request.get_departments(term,amount)
+

```

Comment:
Likewise, you're still missing the bulk of the feature, as right now all this command does is retrieve the data from Banner and nothing else. You need to iterate through `depts` and call some `parse_depts` function, which would then save each department to the database with Django.

It would look something like

```python
def parse_dept(data, term: str):
    dept = Department(code=data['code'], desc=data['desc'], term=term)
    dept.save() # Saves the department to the database
```

Not sure about the actual values(i.e. `data['desc']`) but it should be something along those lines. To check, you should be able to just `print(depts)` in the `scrape_departments` function, then actually run it by typing `python manage.py scrape_departments` when you're in the first `autoscheduler` folder(that has `frontend/`, `scraper/` in it).